server/service/src/sync/translations/indicator_attribute.rs

```diff
@@ -0,0 +1,150 @@
+use anyhow::anyhow;
+use repository::{
+    IndicatorColumnRow, IndicatorLineRow, IndicatorValueType, StorageConnection, SyncBufferRow,
+};
+
+use serde::Deserialize;
+
+use crate::sync::translations::program_indicator::ProgramIndicatorTranslation;
+
+use super::{PullTranslateResult, SyncTranslation};
+
+#[derive(Deserialize, PartialEq)]
+enum LegacyAxis {
+    #[serde(rename = "column")]
+    Column,
+    #[serde(rename = "row")]
+    Row,
+}
+
+#[derive(Deserialize, PartialEq)]
+enum LegacyValueType {
+    #[serde(rename = "number")]
+    Number,
+    #[serde(rename = "string")]
+    String,
+    #[serde(rename = "var")]
+    Var,
+}
+impl Into<Option<IndicatorValueType>> for LegacyValueType {
+    fn into(self) -> Option<IndicatorValueType> {
+        match self {
+            LegacyValueType::Number => Some(IndicatorValueType::Number),
+            LegacyValueType::String => Some(IndicatorValueType::String),
+            LegacyValueType::Var => None,
```

Comment:
Are we later using this to translate to a value table?

Reply:
This `Into` Trait? Yea it's used in the translation mapping below:

https://github.com/msupply-foundation/open-msupply/blob/df69f8618e094dd209fe2f729d5c2ff392437155/server/service/src/sync/translations/indicator_attribute.rs#L88-L109

IMO instead of the mapping being in that function it'd be more rusty to implement `Into` for `OmsRecord = LegacyRecord.into()` but it really doesn't matter haha.